net/shorewall/files/hotplug_iface

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# should restart shorewall when an interface comes up
+
+case "$ACTION" in
+    ifup)
+        /etc/init.d/shorewall restart
+        ;;
+    ifdown)
+        # might need to restore some routing
+        /etc/init.d/shorewall restart
+        ;;
+esac
```

Comment:
Need newline on last line.  Please set your editor's profile to not allow unterminated files.

Reply:
Done